package.json

```diff
@@ -9,8 +9,8 @@
   },
   "type": "module",
   "version": "3.89.0",
-  "module": "lib/main.ts",
-  "types": "lib/main.ts",
+  "module": "dist/weka-ui-components.js",
+  "types": "dist/main.d.ts",
```

Comment:
need to bring this be back to lib:)

Reply:
I did those, but looks like didn't push. Will update in a bit.

Reply:
Reverted.